app/common/data/models.py

```diff
@@ -49,6 +50,7 @@ class User(BaseModel):
     magic_links: Mapped[list["MagicLink"]] = relationship("MagicLink", back_populates="user")
 
     roles: Mapped[list["UserRole"]] = relationship("UserRole", back_populates="user", cascade="all, delete-orphan")
+    collections: Mapped[list["Collection"]] = relationship("Collection", back_populates="created_by")
```

Comment:
Edit: this is the instances a user has created, not the collection schema

Original: I can't anticipate a read model where we'll ever need this back populated when fetching a user - maybe it can't hurt to have it but it would probably be good to be explicit about it being lazy as we'll _always_ need to fetch a user and very rarely care about what collections they've set up